### ISSUE TITLE
Script Error 

### ISSUE THREAD

**Issue by Snebzi** [TreborNamor/Agressive-Bestbuy-Bot#5]
![Bot issue](https://user-images.githubusercontent.com/76721130/103260556-b1e38500-4963-11eb-8066-68b55207e4e8.png)
my bot gives me this error when running, i followed the guide as well as used the updated script.

**Comment by Snebzi** [comment]
![IDK](https://user-images.githubusercontent.com/76721130/103261366-ed338300-4966-11eb-83a6-6480a5bb0776.png)
what does this mean?

**Comment by TreborNamor** [comment]
It looks like you have to upgrade your python version to Python 3. I prefer python 3.9.1 since it's the newest version. You can download the file here.

https://www.python.org/downloads/release/python-391/

**Comment by Snebzi** [comment]
thanks boss

**Comment by EvanPoe** [comment]
> It looks like you have to upgrade your python version to Python 3. I prefer python 3.9.1 since it's the newest version. You can download the file here.
> 
> https://www.python.org/downloads/release/python-391/

I feel like an idiot but... what steps do I take exactly to update python in PyCharm? I've tried downloading the 3.9.1 installer and the embed.zip (which I dropped into the venv folder...) but either approach, both overwriting or starting a 3080bot project from scratch, results in 3.8. Whenever I go to Settings -> Python Interpreter, all it shows me is 3.8.
